docs/JestObjectAPI.md

```diff
@@ -921,6 +921,18 @@ When this API is called, all pending macro-tasks and micro-tasks will be execute
 
 This is often useful for synchronously executing setTimeouts during a test in order to synchronously assert about some behavior that would only happen after the `setTimeout()` or `setInterval()` callbacks executed. See the [Timer mocks](TimerMocks.md) doc for more information.
 
+### `jest.runAllTimersAsync()`
+
+Runs all pending timers until there are none remaining.
+
+Also breaks the event loop, allowing any scheduled promise callbacks to execute _before_ running the timers.
```

Comment:
I know this comes from Sinon docs, but it sounds somewhat dangerous to break the event loop. (Marking the whole line for deletion, because I added this to the comment above already.)

```suggestion
```

Reply:
we should probably call it "yields to the event loop" instead of "breaks"